addons/udes_permissions/views/user_template_view.xml

```diff
@@ -0,0 +1,99 @@
+<odoo>
+    <data>
+        <!-- Form view -->
+        <record id="user_template_form" model="ir.ui.view">
+            <field name="name">user.template.form</field>
+            <field name="model">user.template</field>
+            <field name="arch" type="xml">
+                <form string="User Template">
+                    <sheet>
+                        <!-- Here we have a set of smart buttons that use the 'recursive_child_*' columns
+                        in combination with actions defined on user.template to display the relevant information -->
+                        <div name="button_box" class="oe_button_box">
+                            <button name="action_view_child_templates" type="object" class="oe_stat_button" icon="fa-sitemap">
+                                <field name="family_descendant_count" widget="statinfo" string="Child Templates" />
+                            </button>
+                            <button name="action_view_child_groups" type="object" class="oe_stat_button" icon="fa-users">
+                                <field name="family_group_count" widget="statinfo" string="Implied Groups" />
+                            </button>
+                            <button name="action_view_child_acls" type="object" class="oe_stat_button" icon="fa-list">
+                                <field name="family_acl_count" widget="statinfo" string="Implied ACLs" />
+                            </button>
+                        </div>
+                        <!-- Make name nice and large to edit -->
+                        <div class="oe_title">
+                            <label for="name" class="oe_edit_only" />
+                            <h1>
+                                <field name="name" placeholder="e.g. Stock Manager" />
+                            </h1>
+                        </div>
+                        <!-- Main fields for user template -->
+                        <group>
+                            <group name="basic">
+                                <!-- This field is required at the model level, but we set it to not required
+                                to prevent form validation kicking in before one can be generated and set -->
```

Comment:
Would have thought there'd be a nicer way to get around this by now. The workaround isn't too bad, but does always look a bit odd.